frontend/src/javascript/utils/other.js

```diff
@@ -5,3 +5,7 @@ export function formatDate(date) {
 export function redirectTo(url) {
     window.location.href = url;
 }
+
+export function deepCopy(object) {
```

Comment:
Na razie jest OK, ale ta funkcja może być "niebezpieczna" w przyszłości. Taki deep copy nie poradzi sobie m.in. z:
- datami
- funkcjami
- undefined, Infinity, NaN, etc.
Jeśli będzie używana gdzieś indziej trzeba mieć to na uwadze

Reply:
Zaciągnąłem bibliotekę lodash i skorzystałem z gotowej funkcji cloneDeep, z tego co czytałem tam te problemy powinny być rozwiązane. Osobiście nie korzystałem zbyt dużo z tej biblioteki, ale kojarzę, że to jest całkiem pożyteczna biblioteka i dostarcza wiele przydatnych funkcji także jak coś zachęcam do zapoznania się z nią, moze komuś to nieco ułatwi pisanie kodu w JS.